client/src/Components/CommonComponent/TableParent/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+// import data from './data';
+import './index.css';
+
+const Table = data => {
```

Comment:
same as the other one, the name is misleading, suggesting a generic table component (the 'parent' relationship is also not clear to me). It should be sth like 'StudentTable' or 'StudentList'.